### PR TITLE
Feedback

### PR DESCRIPTION
**Pull request by tavisca-dshrivastav**
@tavisca-nkakade please review

### CODE REVIEW

File: src/Tavisca.TMS.Web/Controllers/FeedbackController.cs [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+        public FeedbackController(IFeedbackService feedbackService)
+        {
+            _feedbackService = feedbackService;
+        }
+        
+        // GET: api/Feedback
+        [HttpGet("session/view")]
+        public async Task<IActionResult> GetFeedbackByTrainee(GetFeedbackRequest request)
+        {
+            var ans = await _feedbackService.GetFeedbackAsync(request);
```
Comment (tavisca-nkakade) [inline src/Tavisca.TMS.Web/Controllers/FeedbackController.cs]: update variable name

File: src/Tavisca.TMS.Core/Validators/FeedbackValidator.cs [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+namespace Tavisca.TMS.Core
+{
+    public class GetFeedbackRequestValidator : AbstractValidator<GetFeedbackRequest>
+    {
+        public GetFeedbackRequestValidator()
+        {
+            RuleFor(x => x.EmployeeId).NotEmpty().MaximumLength(45);
+            RuleFor(x => x.SessionId).NotEmpty().MaximumLength(45);
+            RuleFor(x => x.UserType).InclusiveBetween(1,2);
```
Comment (tavisca-nkakade) [inline src/Tavisca.TMS.Core/Validators/FeedbackValidator.cs]: Use enum for better readbility

File: src/Tavisca.TMS.Tests/Data/Repository/FeedbackRepositoryFixtures.cs [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+        }
+
+        [Fact]
+        public async Task GetFeedback_method_should_return_expected_value_given_data()
+        {
+            var req = new GetFeedbackRequest { EmployeeId = "123", SessionId = "123", UserType = 1 };
+            var res = new List<FeedbackViewDto> { };
+            _feedbackDal.Setup(x => x.GetFeedbackAsync(req)).ReturnsAsync(_feedbackViewDtos);
+            _translator.Setup(x => x.Translate(_feedbackViewDtos)).Returns(feedbacks);
+            var ans = await _feedbackRepository.GetFeedbackAsync(req);
```
Comment (tavisca-nkakade) [inline src/Tavisca.TMS.Tests/Data/Repository/FeedbackRepositoryFixtures.cs]: rename variable

**Comment by tavisca-nkakade** [comment]
Merge PR after resolving commnet

File: src/Tavisca.TMS.Web/Controllers/FeedbackController.cs (hunk 1 above)
Comment (tavisca-dshrivastav) [reply src/Tavisca.TMS.Web/Controllers/FeedbackController.cs]: updated

File: src/Tavisca.TMS.Core/Validators/FeedbackValidator.cs (hunk 2 above)
Comment (tavisca-dshrivastav) [reply src/Tavisca.TMS.Core/Validators/FeedbackValidator.cs]: updated

File: src/Tavisca.TMS.Tests/Data/Repository/FeedbackRepositoryFixtures.cs (hunk 3 above)
Comment (tavisca-dshrivastav) [reply src/Tavisca.TMS.Tests/Data/Repository/FeedbackRepositoryFixtures.cs]: updated

**Pull request merged**